### PR TITLE
Add CLI config page

### PR DESCRIPTION
**Pull request by ZitaNemeckova**
Closes-Issue: AHH-11

Standalone:
Screenshot 2020-11-09 at 17 05 59
Screenshot 2020-11-10 at 09 50 38

Insights:
Screenshot 2020-11-06 at 13 24 23

Screenshot 2020-11-10 at 10 27 03

TODO:
- [x] server URL
- [x] hide snippet for insights
- [x] expand the snippet? Is it needed?

### CODE REVIEW

**Comment by newswangerd** [comment]
You can get rid of the Sync URL. Pulp fixed their URL config.

**Comment by newswangerd** [comment]
Putting the CLI config in the list is really cramped. Is there any reason we can't put it in a modal?

**Comment by ZitaNemeckova** [comment]
@newswangerd Agreed on it with Susan.

File: src/components/repositories/local-repository-table.tsx [hunk 1]
```diff
@@ -124,16 +124,21 @@ export class LocalRepositoryTable extends React.Component<IProps> {
...
-          </ClipboardCopy>
-        </td>
         <td>
           <ClipboardCopy isReadOnly>
             {getRepoUrl(distribution.base_path)}
           </ClipboardCopy>
         </td>
+        {DEPLOYMENT_MODE === Constants.INSIGHTS_DEPLOYMENT_MODE ? null : (
+          <td>
+            <ClipboardCopy isCode isReadOnly variant={'expansion'}>
```
Comment (newswangerd) [inline src/components/repositories/local-repository-table.tsx:134]: This is kind of a mess and also an invalid configuration. This is what I'd recommend

```
    const cliConfig = [
      '[galaxy]',
      `server_list = ${distribution.repository.name}_repo`,
      '',
      `[${distribution.repository.name}_repo]`,
      `url=${getRepoUrl(distribution.base_path)}`,
      'token=<put your token here>',
    ];

[...]

    <ClipboardCopy isCode isReadOnly variant={'expansion'}>
      {cliConfig.join('\n')}
    </ClipboardCopy>
```

File: src/components/repositories/local-repository-table.tsx [hunk 2]
```diff
@@ -71,14 +71,14 @@ export class LocalRepositoryTable extends React.Component<IProps> {
           id: 'updated_at',
         },
         {
-          title: 'Sync URL',
+          title: 'Ansible CLI URL',
```
Comment (newswangerd) [inline src/components/repositories/local-repository-table.tsx:74]: Let's rename this to "Repo URL"

File: src/containers/token/token-insights.tsx [hunk 3]
```diff
@@ -61,10 +62,30 @@ class TokenPage extends React.Component<RouteComponentProps, IState> {
...
             .
           </Section>
+          <Section className='body pf-c-content'>
+            <h2>Server URL</h2>
+            <p>
+              Use this URL to configure Galaxy server in your the{' '}
+              <code>ansible-galaxy</code> client.
+            </p>
+            <ClipboardCopy isReadOnly>
+              {getRepoUrl('').replace('//', '')}
```
Comment (newswangerd) [inline src/containers/token/token-insights.tsx:76]: This is showing up as `https:ci.foo.redhat.com:1337/api/automation-hub/content//`, which is an invalid URL

**Review by newswangerd — CHANGES_REQUESTED** [review]

File: src/containers/token/token-insights.tsx (hunk 3 above)
Comment (newswangerd) [reply src/containers/token/token-insights.tsx:76]: The correct URL should be `https://ci.foo.redhat.com:1337/api/automation-hub/`. Might need to update the `getRepoUrl` function to handle an empty repo string correctly

File: src/components/repositories/local-repository-table.tsx [hunk 4]
```diff
@@ -112,6 +112,15 @@ export class LocalRepositoryTable extends React.Component<IProps> {
   }
 
   private renderRow(distribution) {
+    const cliConfig = [
+      '[galaxy]',
+      `server_list = ${distribution.repository.name}_repo`,
+      '',
+      `[${distribution.repository.name}_repo]`,
```
Comment (newswangerd) [inline src/components/repositories/local-repository-table.tsx:119]: I screwed up. This should be `[galaxy_server.${distribution.repository.name}_repo]

File: src/containers/token/token-insights.tsx (hunk 3 above)
Comment (newswangerd) [inline src/containers/token/token-insights.tsx:76]: this is still showing up as `https:ci.foo.redhat.com:1337/api/automation-hub/`

**Review by newswangerd — CHANGES_REQUESTED** [review]

**Review by newswangerd — APPROVED** [review]

**Pull request merged**